Source/WebCore/rendering/RenderListBox.cpp

```diff
@@ -731,8 +731,7 @@ bool RenderListBox::scrollToRevealElementAtListIndex(int index)
     if (style().isFlippedBlocksWritingMode())
         newOffset *= -1;
 
-    scrollToOffsetWithoutAnimation(scrollbarOrientationForWritingMode(), newOffset);
-
+    scrollToPosition(newOffset);
```

Comment:
Nit: `newOffset` should probably be called `newPosition` now?

Reply:
Well, it's both during this function. Not ideal.